src/wx/visualboyadvance-m.metainfo.xml

```diff
@@ -1,9 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<application>
- <id type="desktop">visualboyadvance-m.desktop</id>
- <developer id="visualboyadvance-m.org">
-   <name>VisualBoyAdvance-M</name>
- </developer>
+<component type="desktop">
+ <id>org.visualboyadvance.visualboyadvance_m</id>
```

Comment:
it's org.visualboyadvance_m.visualboyadvance-m if I recall the formating

Reply:
It's update now, but I need to keep the '_' to maintain appstream compliance.